jsonrpc/debug_endpoint.go

```diff
@@ -295,6 +298,33 @@ func (d *Debug) GetRawTransaction(txHash types.Hash) (interface{}, error) {
 	)
 }
 
+// GetRawReceipts retrieves the binary-encoded receipts of a single block.
+func (d *Debug) GetRawReceipts(filter BlockNumberOrHash) (interface{}, error) {
+	return d.throttling.AttemptRequest(
+		context.Background(),
+		func() (interface{}, error) {
+			header, err := GetHeaderFromBlockNumberOrHash(filter, d.store)
+			if err != nil {
+				return nil, err
+			}
+
+			receipts, err := d.store.GetReceiptsByHash(header.Hash)
+			if err != nil {
+				return nil, err
+			}
+
+			result := make([][]byte, len(receipts))
+
+			for i, receipt := range receipts {
+				b := receipt.MarshalRLP()
+				result[i] = b
```

Comment:
Minor optimization, if you find it necessary (code is still readable enough IMO):

```suggestion
				result[i] = receipt.MarshalRLP()
```